game/players.cc

```diff
@@ -66,23 +56,23 @@ int Players::lookup(std::string const& name) const {
 	return -1;
 }
 
-std::string Players::lookup(int id) const {
-	PlayerItem pi;
-	pi.id = id;
-	auto it = m_players.find(pi);
-	if (it == m_players.end()) return "Unknown Player";
-	else return it->name;
+std::string Players::lookup(PlayerId id) const {
+	const auto it = m_players.find(PlayerItem(id));
+
+    if (it == m_players.end()) 
+        return "Unknown Player";
+
+    return it->name;
 }
 
-void Players::addPlayer (std::string const& name, std::string const& picture, int id) {
+void Players::addPlayer (std::string const& name, std::string const& picture, PlayerId id) {
 	PlayerItem pi;
 	pi.id = id;
 	pi.name = name;
 	pi.picture = picture;
-	pi.path = "";
-
 
-	if (pi.id == -1) pi.id = assign_id_internal();
+	if (pi.id == PlayerItem::UndefinedPlayerId) 
+        pi.id = assign_id_internal();
```

Comment:
Please use braces or correct indent (4 spaces starting from if statement position)

Reply:
OK, I have checked the sources and the indent of line 74 and 75 is correct, but they are different. While 74 uses the original tab line 75 uses 8 spaces for indent. What do you suggest in this case?

Reply:
Ahh i see, the one is a tab the other just spaces. Uhm indent the line 75 with 2 tabs instead of one :)

Sometime later we'll have an editorconfig available to force such things ;P